enterprise/internal/campaigns/service_test.go

```diff
@@ -507,6 +508,8 @@ func TestService_UpdateCampaignWithNewCampaignPlanID(t *testing.T) {
 		wantModified repoNames
 		// Repositories for which we want to create a new ChangesetJob (and thus a Changeset)
 		wantCreated repoNames
+		// An error to be thrown when attempting to do the update
+		wantErr *error
```

Comment:
```suggestion
		wantErr error
```

You don't need to make this a pointer. Instead, if it's an `error`, you can do this in the body of the test:

```go
				if have, want := fmt.Sprint(err), tc.wantErr; have != want {
					t.Errorf("error:\nhave: %q\nwant: %q", have, want)
				}
```

See here: https://play.golang.org/p/ACwYRofWHgK

Reply:
Hmm I could not figure out how to properly check

```go
if tt.wantErr != nil {
	return
}
```
because after erroring here, it makes no sense to further continue within that test ..

Reply:
If it's an `error` you can still check whether it's `nil` or not:

```go
if err != nil {
  return err
}
```